src/amo/routes.js

```diff
@@ -1,85 +0,0 @@
-import config from 'config';
-import * as React from 'react';
-import { IndexRoute, Route } from 'react-router';
-
-import About from 'amo/components/StaticPages/About';
-import Addon from 'amo/components/Addon';
-import AddonReviewList from 'amo/components/AddonReviewList';
-import App from 'amo/components/App';
-import CategoriesPage from 'amo/components/CategoriesPage';
-import Category from 'amo/components/Category';
-import Collection from 'amo/components/Collection';
-import CollectionEdit from 'amo/components/CollectionEdit';
-import CollectionList from 'amo/components/CollectionList';
-import Home from 'amo/components/Home';
-import LandingPage from 'amo/components/LandingPage';
-import LanguageTools from 'amo/components/LanguageTools';
-import SearchTools from 'amo/components/SearchTools';
-import NotAuthorized from 'amo/components/ErrorPage/NotAuthorized';
-import NotFound from 'amo/components/ErrorPage/NotFound';
-import ReviewGuide from 'amo/components/StaticPages/ReviewGuide';
-import SearchPage from 'amo/components/SearchPage';
-import ServerError from 'amo/components/ErrorPage/ServerError';
-import UserProfile from 'amo/components/UserProfile';
-import UserProfileEdit from 'amo/components/UserProfileEdit';
-import SimulateAsyncError from 'core/components/error-simulation/SimulateAsyncError';
-import SimulateClientError from 'core/components/error-simulation/SimulateClientError';
-import SimulateSyncError from 'core/components/error-simulation/SimulateSyncError';
-
-// If you add a new route here, check that the nginx rules maintained by ops
```

Comment:
Could you put this comment back? It's still important.

Reply:
ah right, I forgot.